samples/istio/quickstart/fortio.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: fortio
+spec:
+  template:
+    spec:
+      volumes:
+      - name: shared
+        emptyDir: {}    
+      containers:
+      - name: fortio
+        image: fortio/fortio
+        command:
```

Comment:
Using the square bracket notation as in line 34 will shorten this deployment significantly.